libmscore/system.cpp

```diff
@@ -1103,7 +1103,7 @@ void System::scanElements(void* data, void (*func)(void*, Element*), bool all)
 qreal System::staffYpage(int staffIdx) const
       {
       if (_staves.size() <= staffIdx || staffIdx < 0) {
-            qFatal("staffY: staves %d: bad staffIdx %d", _staves.size(), staffIdx);
+            qWarning("staffY: staves %d: bad staffIdx %d", _staves.size(), staffIdx);
```

Comment:
I think, need to find and fix the reason why staffIdx is not valid  
And in this place add assert

Reply:
like the next commit?

Reply:
Yes :)   
But there is a problem, why is an invalid index passed?

Reply:
I guess because there is no staff, just a frame

Reply:
Then I think this method should not be called, in such cases.